processing file asmFiles/test.beqnottaken.asm (1 of 25)
processing file asmFiles/test.beqtaken.asm (2 of 25)
processing file asmFiles/test.bgenottaken.asm (3 of 25)
processing file asmFiles/test.bgetaken.asm (4 of 25)
processing file asmFiles/test.bgeunottaken.asm (5 of 25)
processing file asmFiles/test.bgeutaken.asm (6 of 25)
processing file asmFiles/test.bltnottaken.asm (7 of 25)
processing file asmFiles/test.blttaken.asm (8 of 25)
processing file asmFiles/test.bltunottaken.asm (9 of 25)
processing file asmFiles/test.bltutaken.asm (10 of 25)
processing file asmFiles/test.bnenottaken.asm (11 of 25)
processing file asmFiles/test.bnetaken.asm (12 of 25)
processing file asmFiles/test.branchjump1.asm (13 of 25)
processing file asmFiles/test.controlflowhazard.asm (14 of 25)
processing file asmFiles/test.datahazard.asm (15 of 25)
processing file asmFiles/test.forwardinghazard.asm (16 of 25)
processing file asmFiles/test.halt.asm (17 of 25)
processing file asmFiles/test.jal.asm (18 of 25)
processing file asmFiles/test.jalr.asm (19 of 25)
processing file asmFiles/test.ldst1.asm (20 of 25)
processing file asmFiles/test.ldst2.asm (21 of 25)
processing file asmFiles/test.loadstore.asm (22 of 25)
processing file asmFiles/test.lui.asm (23 of 25)
processing file asmFiles/test.rtype.asm (24 of 25)
--- memsim.hex	2025-02-18 21:27:18.656839423 -0500
+++ memcpu.hex	2025-02-18 21:27:18.638838736 -0500
@@ -27,12 +27,12 @@
 :04001A00007AAE2397
 :04001B0003CAA02351
 :04001C00FFFFFFFFE4
-:0401C10000001261C7
+:0401C10000000261D7
 :0401C2000000000930
-:0401C30000010A5AD3
+:0401C30000000A5AD4
 :0401C40000000740F0
-:0401C500FFFFDA70EE
-:0401C600000037F806
-:0401C7000000D1560D
-:0401C800000D269070
+:0401C500FFFFFA70CE
+:0401C600000007F836
+:0401C70000000156DD
+:0401C800000026907D
 :00000001FF
processing file asmFiles/test.structuralhazard.asm (25 of 25)
